pkg/metrics/metrics.go

```diff
@@ -0,0 +1,12 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"net/http"
+)
+
+// ServeMetrics exposes metrics in Prometheus format
+func ServeMetrics() {
+	http.Handle("/metrics", promhttp.Handler())
+	http.ListenAndServe(":2112", nil)
```

Comment:
@Surbhidongaonkar We should read port from env var. Something like `METRICS_PORT`. And pass that env variable through deployment spec (like https://github.com/infracloudio/botkube/blob/develop/helm/botkube/templates/deployment.yaml#L41)